restapi/src/main/java/io/stargate/web/docsapi/dao/Paginator.java

```diff
@@ -24,11 +39,19 @@ public Paginator(String pageState, int pageSize) {
     docPageSize = pageSize;
 
     if (pageState != null) {
-      byte[] decodedBytes = Base64.getDecoder().decode(pageState);
+      byte[] decodedBytes = decode(pageState);
       this.currentPageState = ByteBuffer.wrap(decodedBytes);
     }
   }
 
+  public static byte[] decode(String pageState) {
+    return Base64.getUrlDecoder().decode(pageState);
```

Comment:
Do we need to temporarily decode both encodings? Wondering if when this rolls out it could break inflight requests.

Reply:
Good point.

An old paging state with a `+` would be corrupted at the HTTP level, it reaches our code already broken.

However, old paging state that only contain `/` chars (the seconds and last difference for the [URL-safe scheme](https://www.ietf.org/rfc/rfc4648.html#section-5)) should be reusable. I'll update.

Reply:
I think we can un-break old paging state too... :hourglass:

Reply:
I think I'll include plain REST in this PR too... the problem is the same in both code areas.

Reply:
updated, force pushed